src/test/java/org/jenkinsci/plugins/github_branch_source/GitHubApp.java

```diff
@@ -0,0 +1,51 @@
+package org.jenkinsci.plugins.github_branch_source;
+
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import hudson.util.Secret;
+
+public class GitHubApp {
+
+    // PKCS8 private key (https://stackoverflow.com/a/22176759/4951015)
+    private static final String PRIVATE_KEY = "-----BEGIN PRIVATE KEY-----\n"
+            +
```

Comment:
BTW when updating to 2.479.x we can switch to a multiline text block which would be more legible.